lua/catppuccin/groups/integrations/lsp_saga.lua

```diff
@@ -24,42 +25,89 @@ function M.get()
 		DiagnosticSource = { fg = C.overlay0 },
 		DiagnosticPos = { fg = C.surface2 },
 		DiagnosticWord = { fg = C.text },
+		DiagnosticHead = { fg = C.blue },
 		CallHierarchyIcon = { fg = C.mauve },
 		CallHierarchyTitle = { fg = C.blue },
 		SagaShadow = { bg = C.crust },
 		OutlineIndent = { fg = C.overlay2 },
 
-		-- Because kind never worked after changing colorscheme
-		LspSagaWinbarFile = { fg = C.text },
-		LspSagaWinbarModule = { fg = C.blue },
-		LspSagaWinbarNamespace = { fg = C.peach },
-		LspSagaWinbarPackage = { fg = C.mauve },
-		LspSagaWinbarClass = { fg = C.mauve },
-		LspSagaWinbarMethod = { fg = C.mauve },
-		LspSagaWinbarProperty = { fg = C.sky },
-		LspSagaWinbarField = { fg = C.teal },
-		LspSagaWinbarConstructor = { fg = C.blue },
-		LspSagaWinbarEnum = { fg = C.green },
-		LspSagaWinbarInterface = { fg = C.peach },
-		LspSagaWinbarFunction = { fg = C.mauve },
-		LspSagaWinbarVariable = { fg = C.blue },
-		LspSagaWinbarConstant = { fg = C.sky },
-		LspSagaWinbarString = { fg = C.green },
-		LspSagaWinbarNumber = { fg = C.green },
-		LspSagaWinbarBoolean = { fg = C.peach },
-		LspSagaWinbarArray = { fg = C.blue },
-		LspSagaWinbarObject = { fg = C.peach },
-		LspSagaWinbarKey = { fg = C.red },
-		LspSagaWinbarNull = { fg = C.red },
-		LspSagaWinbarEnumMember = { fg = C.green },
-		LspSagaWinbarStruct = { fg = C.mauve },
-		LspSagaWinbarEvent = { fg = C.mauve },
-		LspSagaWinbarOperator = { fg = C.green },
-		LspSagaWinbarTypeParameter = { fg = C.green },
-		LspSagaWinbarTypeAlias = { fg = C.green },
-		LspSagaWinbarParameter = { fg = C.blue },
-		LspSagaWinbarStaticMethod = { fg = C.peach },
-		LspSagaWinbarMacro = { fg = C.red },
+		-- Lspkind icons support
+		LspKindClass = { fg = C.yellow },
```

Comment:
I mean we directly set it like this instead:

```lua
SagaWinbarClass = { link = "Class }
```

Reply:
Maybe they can be changed to:
```lua
LspKindClass = { link = "Class" },
```
?